script/parse-examples

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+for name in elasticsearch guava RxJava; do
+  repo=examples/$name
+
+  # TODO: Consider cloning example repos instead of using submodules.
```

Comment:
@maxbrunsfeld, @aymannadeem - What do you think about converting this repository over to the cloning approach (instead of submodules)?

Reply:
:+1: Yeah, that sounds like a good change.

Reply:
I'm cool with that.